docs/specs/openapi.json

```diff
@@ -1,7 +1,7 @@
 {
     "openapi": "3.0.0",
     "info": {
-        "description": "The API for Cost Management",
+        "description": "Access your cloud and OpenShift data you integrated with cost management.",
         "version": "1.0.2",
```

Comment:
Could you also set this version to `v1`? I have no idea where 1.0.2 came from, and given that our api path is api/cost-management/v1, it feels like this version should also be v1.

Reply:
I changed to v 1 with no dots. Does it need to be `1.0.0`?

I'm editing the right openapi file right? The only other ones are in /koku/sources and /koku/masu and look like they do different stuff.

Reply:
I think it should be `v1`. And yes, this is the correct doc to edit.

Reply:
There seems to be zero consistency, but looking at this one:
https://console.redhat.com/docs/api/compliance
it uses `v1`.

Reply:
Ok, updated to "v1"